models/ht_neuron.cpp

```diff
@@ -83,14 +82,29 @@ ht_neuron_dynamics( double, const double y[], double f[], void* pnode )
   // Synaptic channels
   double I_syn = 0;
 
+  /*
+   * NMDA conductance
+   *
+   * We need to take care to handle instantaneous blocking correctly.
+   * If the unblock-variables Mg_{fast,slow} > Mg_ss, the steady-state
```

Comment:
This looks unclear to me, you mean "if the unblock variables are greater than their respective steady-state values", right? I think the combination of equation and text sentence with commas is rather disturbing.